md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
+
+Be kind to the plants and animals.
```

Comment:
I prefer "discouraged" over "prohibited", not sure why that language bothers me exactly I think we can't really prohibit anything unless it's against the law?  

I think being grumpy at other people is the definition of grumpy, otherwise you are just upset and nobody is around.  Once people are around you then you project your mental state on them and become  Señor Grumpypants.  Which amongst friends is ok, but it's not the more productive role to be in a professional environment or to get stuff done, but people deal with it like any barrier to success.  We should encourage people to play the role of Big Bird, not Oscar, when you encounter a grumpy peer.